lib/rules/dot-notation.js

```diff
@@ -142,7 +142,7 @@ module.exports = {
                     !allowKeywords &&
                     !node.computed &&
                     node.property.type === "Identifier" &&
-                    keywords.indexOf(String(node.property.name)) !== -1
+                    keywords.includes(String(node.property.name))
```

Comment:
`Identifier.name` is always a string.

Reply:
Ok, good to know, but I think for another PR.